Added README for network tokens

nevelis opened the pull request.

Commit:
Added README for network tokens

yiannisy reviewed (changes requested):
Can you add a section with instructions for the reader on how to download a client (Jitsi?) and make a call that uses a token?

Commit:
README update

Commit:
Added missing pieces, removed debug tokens

Commit:
Fixed missing parenthesis

Commit:
Encode the network token as binary

Commit:
Use another function for hex decoding

The absl one produced garbage, and hex_encode() is used elsewhere in
this file.

Commit:
README update, notes on Jitsi D/L & installation

yiannisy commented:
I don't think we should require people to use adb. Most likely user would be a network engineer that has no Android development setup, and wants to get a network trace or test an E2E workflow in their network. 

Anything blocking us from producing a proper signed version that they can just install?

Commit:
Token reflection fix

Only reflect the token if requested, and put the correct flag on the
reflected token.

Commit:
Removed more temporary tokens

Commit:
Readme update with pcap link
